src/App.tsx

```diff
@@ -15,36 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortField {
+  alphabet = 'alphabet',
+  length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortingType: SortField | null,
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sortingType) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortingType) {
+        case SortField.alphabet:
+          return good1.localeCompare(good2);
+        case SortField.length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return preparedGoods;
+}
```

Comment:
[CODE STYLE] - The function's name should start with a verb. In this case, the function `getPreparedGoods` is more appropriate to be named `prepareGoods`.